stack/adding-a-new-package.rst

```diff
@@ -16,21 +16,29 @@ DM packaging of third party code should proceed as described in :doc:`packaging-
 If the new package needs a distinct Jira component (most will), any DMLT member (such as your manager) can add one.
 
 Adding a package to a distributed product
------------------------------------------
+=========================================
+
+RFC process
+-----------
 
 New packages intended for distribution to end users should generally be added as a dependency of a "top-level product:" these are the roots of the LSST package hierarchy.
 They include ``lsst_apps``, ``lsst_distrib``, ``qserv_distrib`` and ``lsst_sims``.
 
-Before adding a new dependency one these top-level distribution products, it must be approved through the :doc:`RFC process </processes/decision_process>`.
+Before adding a new dependency to one these top-level distribution products, it must be approved through the :doc:`RFC process </communications/rfc>`.
 Consensus must be reached regarding both the name and the suitability of the new package.
 Before adopting the RFC, the following steps must be completed:
 
 * Implementation tickets are created to cover package creation.
-* The package is migrated to the ``lsst`` org, if not already there.
+* The package is migrated to the `LSST organization on GitHub`_, if not already there.
+* The package must have a descriptive readme file at the root level, describing how it fits into the Science Pipelines infrastructure.
+* The package docs must build and be linked from a branch of the `Science Pipelines docs`_.
```

Comment:
A package RFC can be adopted before any code exists.  How can we insist that the docs must already exist?

I think the most you can do is say that this must be part of the implementation tickets mentioned in the first bullet, and that the docs must build prior to marking the RFC as implemented.

Reply:
Can a package RFC for "add this to lsst_distrib" really be adopted before any code exists? Can it even really be written, since you can't say anything about how it actually connects to the rest of the ecosystem. Even if that's theoretically possible, I'm not aware of us ever done an "add a package that hasn't been created yet to the build" RFC.

Reply:
You can say what it's intended to do.  As a sort-of example, the drp_pipe RFC (775) didn't exactly have code for all of the packages before it was adopted.